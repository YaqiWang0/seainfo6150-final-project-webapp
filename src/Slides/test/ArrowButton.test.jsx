import React from "react";
import ArrowButton from "../ ArrowButton";

describe("ArrowButton test", () => {

    it('render successfully', function () {
        const {container} = render(
            <ArrowButton />
        );
        expect(container).toMatchSnapshot();
    })
})