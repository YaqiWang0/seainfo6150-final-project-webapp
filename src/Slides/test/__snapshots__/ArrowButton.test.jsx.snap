// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowButton test render successfully 1`] = `
<div>
  <a
    class="arrow_left"
    href="#"
  />
</div>
`;
